### ISSUE TITLE
Any way to get the tests working on travis/CRAN?

### ISSUE THREAD

**Issue by daattali** [MangoTheCat/shinytest#19]
CRAN is maybe not super important, but since the entire focus of this package is testing, it'd be great if I could add tests for my shiny apps that could run on travis instead of relying on myself to manually run them.

I suppose in order for that to work, the build would need to install phantomjs and set it in the PATH. If that can be done (and it should be doable? But I've never dealt with building custom software on R package builds so I don't know how) then the tests should be able to run smoothly I'd imagine.

But maybe it's way harder than that.

**Comment by gaborcsardi** [comment]
Travis: I have started already, I need `webdriver` first: https://github.com/MangoTheCat/webdriver/blob/travis/.travis.yml#L8-L12

It fails because a process gets stuck. This is very hard to debug, but I agree it is critical to fix. I wrote a package just to manage the processes (`processx`) and it works in general, but fails here.....

I also agree that we need a good easy way to ensure that `phantomjs` is available. An `install_phantomjs` function, of maybe even just a `phantomjs` R package that we can just depend on.
